### PR TITLE
Redirect to collaboration page and clean up

### PR DESCRIPTION
**Pull request by yfg-dev**

### CODE REVIEW

File: frontend/Peerprep/pages/index.vue [hunk 1]
```diff
@@ -294,23 +287,16 @@ onUnmounted(() => {
...
               :disabled="isProcessing || matchFound || collaborationStore.isCollaborating">
               Match
             </Button>
-            -->
+            <Button type="button" @click="collaborationStore.clearCollaborationInfo">Clear</Button>
+
 
-            <Button v-else class="w-3/4 mt-3" :disabled="isProcessing">
-              Match
-            </Button>
```
Comment (Huggenguggen) [inline frontend/Peerprep/pages/index.vue:306]: image
Not sure what's the issue but UI has been broken

**Review by Huggenguggen — CHANGES_REQUESTED** [review]
Other than the UI issue, all seems good!

File: frontend/Peerprep/pages/index.vue (hunk 1 above)
Comment (yfg-dev) [reply frontend/Peerprep/pages/index.vue:306]: fixed the position of the clear button

File: frontend/Peerprep/components/navigation/Navbar.vue [hunk 2]
```diff
@@ -29,7 +29,7 @@ const getInitials = () => {
                 <div class="flex items-center space-x-6">
                     <NavLink to="/" exact>Home</NavLink>
                     <NavLink to="/questions">Questions</NavLink>
-                    <!-- <NavLink to="/collaboration">Collaboration</NavLink> -->
+                    <NavLink to="/collaboration">Collaboration</NavLink>
```
Comment (Huggenguggen) [inline frontend/Peerprep/components/navigation/Navbar.vue]: I think we should still not allow users to navigate to collaboration via navbar if they are being redirected via a match

**Review by Huggenguggen — CHANGES_REQUESTED** [review]
Just a few small things:
1. Please change the dockerfile used for ui in docker-compose to the production version
2. collaboration page should not be in navbar

File: frontend/Peerprep/components/navigation/Navbar.vue (hunk 2 above)
Comment (yfg-dev) [reply frontend/Peerprep/components/navigation/Navbar.vue]: removed link to collaboration from navbar and added redirect from /collaboration to / when collabstore is empty is middleware

**Review by Huggenguggen — APPROVED** [review]
LGTM

**Pull request merged**